### PR TITLE
Add serverless-discovery-plugin to the plugin list

### PR DESCRIPTION
**Pull request by aregier**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +84,7 @@ After adding your plugin to the `plugins.json` file, run `npm run docs` to regen
...
 | **[Ephemeral - `serverless-ephemeral`](https://github.com/Accenture/serverless-ephemeral)** <br/> by [Accenture](http://github.com/Accenture) <br/> Build and include custom stateless libraries for any language | ![Github Stars](https://img.shields.io/github/stars/Accenture/serverless-ephemeral.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-ephemeral.svg?label=Downloads&style=for-the-badge)|
-| **[Es Logs - `serverless-es-plugin`](https://github.com/daniel-cottone/serverless-es-logs)** <br/> by [daniel-cottone](http://github.com/daniel-cottone) <br/> A Serverless plugin to transport logs to ElasticSearch | ![Github Stars](https://img.shields.io/github/stars/daniel-cottone/serverless-es-logs.svg?label=Stars&style=for-the-badge) <br/> ![NPM Downloads](https://img.shields.io/npm/dt/serverless-es-plugin.svg?label=Downloads&style=for-the-badge)|
```
Comment (eahefnawy) [inline README.md]: Why did we change this?

Comment (aregier) [reply README.md]: Hmm. That's interesting. I just ran `npm run docs` and included the exact output here. Is it possible the json and the markdown have gotten out of sync as part of a different pull request?

I'm assuming I shouldn't make any manual changes to the markdown whatsoever after running the command - please let me know if that is in error.

Comment (eahefnawy) [reply README.md]: hmm yeah something might have got messed up there. Could you rebase master and try again?

**Comment by aregier** [comment]
@eahefnawy Sorry for the delay, I got sidetracked with some other things. I rebased this morning, and it seems to have taken care of the issue.

**Comment by eahefnawy** [comment]
Thanks @aregier 😊

**Pull request merged**